webdrivermanager upgrade

QAonTheAir opened the pull request:
support download chrome driver for chrome version 116 or newer

QAonTheAir committed:
webdrivermanager upgrade to 5.4.1

support download chrome driver for chrome version 116 or newer

duyluonganh commented:
@hieuln1991 Did you test from your side? Can you update the dependency to the latest version?

QAonTheAir committed:
Update pom.xml

update webdrivermanager to 5.5.3

raidanny commented:
the issue is still with latest webdrivermanager v5.5.3, I tested it, doesnt work by updating webdrivermanager version  https://github.com/bonigarcia/webdrivermanager/issues/1137

duyluonganh commented:
@raidanny Thanks for your info. We will look at it carefully. If you are using our Katalium Server, you can refer to my workaround here: https://forum.katalon.com/t/katalium-a-workaround-to-let-katalium-server-download-chromedriver-116-or-later-versions/99010
cc: @hieuln1991

perepelovns commented:
On vm, on linux Ubuntu, we have some problems too
wdm exception

io.github.bonigarcia.wdm.config.WebDriverManagerException at WebDriverManager.java:1760
        Caused by: java.lang.RuntimeException at ApacheDockerHttpClientImpl.java:195
            Caused by: java.io.IOException at DomainSocket.java:63
                Caused by: com.sun.jna.LastErrorException at LinuxDomainSocket.java:-2

perepelovns commented:
Help chang library version selenium
Need change ALL library versions selenium

DON'T work since 4.14.*

duyluonganh commented:
I think the issue was resolved in WDM v5.6.2. Is that correct @raidanny, @hieuln1991  ?

QAonTheAir committed:
Update dependency version

QAonTheAir commented:
> I think the issue was resolved in WDM v5.6.2. Is that correct @raidanny, @hieuln1991 ?

yes. I updated WDM to 5.6.2 ( also updated some dependencies then packaged a new kata-server and it works well.
